zcash_client_backend/src/data_api.rs

```diff
@@ -2372,6 +2374,24 @@ pub trait WalletWrite: WalletRead {
         request: Option<UnifiedAddressRequest>,
     ) -> Result<Option<UnifiedAddress>, Self::Error>;
 
+    /// Generates, persists, and marks as exposed a diversified address for the specified account
+    /// at the provided diversifier index. If the `request` parameter is `None`, an address should
+    /// be generated using all of the available receivers for the account's UFVK.
+    ///
+    /// In the case that the diversifier index is outside of the range of valid transparent address
+    /// indexes, no transparent receiver should be generated in the resulting unified address. If a
```

Comment:
```suggestion
    /// indices, no transparent receiver should be generated in the resulting unified address. If a
```
(We use "indices" far more often than "indexes"; the latter is only used a couple of times in `zcash_transparent/src/keys.rs`.)